spring-pulsar/src/main/java/org/springframework/pulsar/core/PulsarAdministration.java

```diff
@@ -0,0 +1,171 @@
+package org.springframework.pulsar.core;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.logging.LogFactory;
+import org.apache.pulsar.client.admin.PulsarAdmin;
+import org.apache.pulsar.client.admin.PulsarAdminBuilder;
+import org.apache.pulsar.client.admin.PulsarAdminException;
+import org.apache.pulsar.client.api.PulsarClientException;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.SmartInitializingSingleton;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.core.log.LogAccessor;
+import org.springframework.util.CollectionUtils;
+
+public class PulsarAdministration implements ApplicationContextAware, SmartInitializingSingleton, PulsarAdminOperations {
```

Comment:
To follow on - I realized that the pulsar admin client is named `PulsarAdmin` so you chose this path to avoid a duplilcate name only differentiated by packages - which I thank you for. having to fully qualify in the code is not a pretty thing. I am ok w/ this name now. Please close this thread once you digest it.